Block/Config.php

```diff
@@ -99,4 +99,30 @@ public function getLocale()
     {
         return $this->localeResolver->getLocale();
     }
+
+    /**
+     * @return string
+     */
+    public function getProductSku()
+    {
+        $product = $this->getProduct();
+        $children = $this->config->getUseChildSku() ? $product->getTypeInstance()->getUsedProducts($product) : [];
+
+        return count($children) > 0 ? array_values($children)[0]->getSku() : $product->getSku();
```

Comment:
It shouldn't get to that point because the conditional count($children) > 0 would return false and the false option will be run instead.